packages/x-license/src/useLicenseVerifier/useLicenseVerifier.ts

```diff
@@ -47,15 +48,15 @@ export function useLicenseVerifier(
       return sharedLicenseStatuses[packageName]!.licenseVerifier;
     }
 
-    const acceptedScopes: LicenseScope[] = packageName.includes('premium')
-      ? ['premium']
-      : ['pro', 'premium'];
+    const acceptedScopes = extractAcceptedScopes(packageName);
```

Comment:
I would maybe rename this:

```suggestion
    const acceptedPlans = extractAcceptedPlans(packageName);
```

The use of "scopes" could be confused with "plans".

Reply:
Fixed in #13619